### ISSUE TITLE
use_g_autoptr_inline_cleanup: False positive on multiple assignations

### ISSUE THREAD

**Issue by A6GibKm** [bilelmoussaoui/gobject-lint#19]
The following:

```c
static void
example (void)
{
  g_autoptr(GFile) dest_file = NULL;

  dest_file = g_file_new_for_uri ("foo");
  g_clear_object (&dest_file);
  dest_file = g_file_new_for_uri ("bar");

  // ...
}
```

Produces the following false-positive:
```
                             gtr_dl_teams_parse_file_info,
                             self);
  ...
  static void
  example (void)
  ...
  >  g_autoptr(GFile) dest_file = NULL;
  ...
    dest_file = g_file_new_for_uri ("foo");
    g_clear_object (&dest_file);
  error: Consider using g_autoptr(g_autoptr(GFile)) dest_file to avoid manual cleanup
  rule: use_g_autoptr_inline_cleanup
```

**Comment by bilelmoussaoui** [comment]
Should be better now
